sos/plugins/openvswitch.py

```diff
@@ -43,20 +75,16 @@ def setup(self):
         ])
 
         self.add_cmd_output([
-            # The '-s' option enables dumping of packet counters on the
-            # ports.
-            "ovs-dpctl -s show",
-            # Capture the in-kernel flow information if it exists
-            "ovs-dpctl dump-flows -m",
-            # Capture the flow information also for offloaded rules
-            "ovs-dpctl dump-flows type=offloaded -m",
```

Comment:
It's not clear to me why these are getting dropped with this commit.

Reply:
These commands were were removed from the master with this commit eb145d9484cae850f6fc4984e2e229486241c833

I believe that if we put them back here, we should put them back on master, but I'll let you judge.

Reply:
> This commit does a lot more than just add the 6wind commands. It seems to try to backport the entire `master` copy of the plugin, but it gets a bit mangled in the process. There's bits of the old API bleeding through, and there are changes being made that are unrelated to 6wind.
> 
> For the purpose of this commit, this should be exclusively focused on adding the 6wind commands and nothing else. If there are other issues with the `legacy-3.9` version of this plugin then separate commits should correct and document those.

Just wondering if it's an issue to sync up master's version with legacy? I thought this was a good idea.

Reply:
Well, the main thing is sync'ing the plugin with `master` is entirely separate from adding the 6wind commands, which is what this commit documents it does. If a sync is desired, then it should be handled in a commit that explicitly highlights that it is syncing the legacy version to the current master, and why that is needed.

Then there's the matter of determining if the dropped commands are indeed not useful for any system that might install a packaged version of sos from the `legacy-3.9` branch.

Reply:
That makes sense. I re-added them. We should consider to put them back on the master branch actually, this is lacking.